docker-compose.yml

```diff
@@ -93,52 +136,22 @@ services:
     volumes:
       - ./conf/portal/settings_secret.py:/srv/www/portal/server/portal/settings/settings_secret.py
       - ./conf/portal/settings_custom.py:/srv/www/portal/server/portal/settings/settings_custom.py
+      - /var/www/portal/portal/media:/srv/www/portal/server/media
```

Comment:
are these to volumes required in the workers?

Reply:
Correct, I have no idea where I got this from =)

Reply:
I think from the diff.  but I do see that the secrets are potentially needed if we want to test all features of Core locally